lib/modules/datasource/types.ts

```diff
@@ -88,6 +88,7 @@ export interface ReleaseResult {
   replacementVersion?: string;
   lookupName?: string;
   packageScope?: string;
+  datasourceProvidedReleaseNotes?: string | undefined;
```

Comment:
Commenting to make sure we have an answer before we merge. The maintainers will resolve this comment later.

Please put your answer in the _issue_. 😉 

`@rarkins` asked:

> Can you clarify one simple/quick question to get started: In cases where release notes are included on the registry, are they contained in the data (i.e. API responses) which Renovate already retrieves using the nuget datasource, or would this be new endpoints on the nuget registry which Renovate isn't currently querying?

- https://github.com/renovatebot/renovate/issues/14128#issuecomment-2259729409

Reply:
@HonkingGoose this is answered already by the early work I've done in this draft PR, I believe. I'm extracting the data from the nuspec file that's already been pulled in order to be a potential source of the repository URL. The goal of adding this variable is to be able to assign the data at that point and pass it through the process where I hope it can be used as a fallback in the case other higher-fidelity release note sources aren't found.